azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/QueueDescription.java

```diff
@@ -81,23 +82,24 @@ public void setLockDuration(Duration lockDuration)
     }
 
     /**
-     * Gets the maximum size of the queue in megabytes, which is the size of memory allocated for the queue.
+     * @return Gets the maximum size of the queue in megabytes, which is the size of memory allocated for the queue.
```

Comment:
>@return Gets the m [](start = 7, length = 18)

Again, we should have both method description and returns.

Reply:
As discussed, not duplicating the comments for getters-setters

---
In reply to: [260571040](https://github.com/Azure/azure-service-bus-java/pull/341#discussion_r260571040) [](ancestors = 260571040)